test/database.js

```diff
@@ -169,17 +169,6 @@ describe('Database', function() {
       assert.strictEqual(database.pool_.calledWith_[1], POOL_OPTIONS);
     });
 
-    it('should re-emit SessionPool errors', function(done) {
```

Comment:
I know generic-pool does not emit generic error events (I don't think at least). It *does* appear to emit `factoryCreateError` and `factoryDestroyError`, I think it would be in the users interest to emit these errors.